contracts/AllowedStrategy.sol

```diff
@@ -0,0 +1,270 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.11;
+
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+import "./common/UsingRegistryUpgradeable.sol";
+import "./common/UUPSOwnableUpgradeable.sol";
+import "./interfaces/IAccount.sol";
+import "./interfaces/IGroupHealth.sol";
+import "./interfaces/IManager.sol";
+import "./Managed.sol";
+
+contract AllowedStrategy is UUPSOwnableUpgradeable, UsingRegistryUpgradeable, Managed {
```

Comment:
From the name I was under the impression this was a contract managing all allowed strategies. But reading through the code, seems like it's more about the "specific group" strategy? If so, some of my comments might be irrelevant, but this should then probably be rename (`SpecificGroupStrategy`?).

Reply:
That is my misunderstanding from previous comments - renamed to "SpecificGroupStrategy"